tests/integration/test_vulnerability_detector/test_feeds/test_download_feeds.py

```diff
@@ -10,15 +10,16 @@
 from wazuh_testing.tools import LOG_FILE_PATH
 from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
 from wazuh_testing.tools.monitoring import FileMonitor
-from wazuh_testing.vulnerability_detector import make_vuln_callback, VULN_DETECTOR_GLOBAL_TIMEOUT, clean_vd_tables
+from wazuh_testing.vulnerability_detector import make_vuln_callback, VULN_DETECTOR_GLOBAL_TIMEOUT, \
+                                                 clean_vuln_and_sys_programs_tables
```

Comment:
```suggestion
import wazuh_testing.vulnerability_detector as vd
```

Reply:
Done in 013ae9e573cd44ee4c669aaa0293b5bac6585505